grai-server/docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
       - DJANGO_SUPERUSER_PASSWORD=super_secret
       - SERVER_HOST=localhost
       - FRONTEND_HOST=localhost
+      - SECRET_KEY=secret-key
```

Comment:
Were you hitting an error without a `SECRET_KEY` defined? If so there may be a bug somewhere, docker-compose should generate a persistent key if no env variable is provided.

Reply:
The key wasn't persistent, so I was having to log in afresh each time I recreated the container.

Reply:
The DBT integration PR now includes refactored django settings so that we can operate dev & prod separately. You can use the dev configuration (with a static secret key) by setting the `DJANGO_SETTINGS_MODULE` environment variable to `the_guide.settings.dev`. 

Maybe we can move the `examples` folder under the grai-server repository into the root so those are the canonical "prod" deployments and the docker-compose here is just for development purposes?